pandas/core/generic.py

```diff
@@ -9048,9 +9048,9 @@ def where(
         try_cast : bool, default False
             Try to cast the result back to the input type (if possible).
 
-        Returns
+        Returns or None
```

Comment:
I guess that this ``or None`` is unnecessary.

Reply:
Oh, this is a mistake. Thank you.